### PR TITLE
Installer script with easy configuration for noobs

### PR DESCRIPTION
**Pull request by helloitu**
Short Description: 
This script when executed , configure the bot , compile encrypt.so , fix all the bugs , and have a easy config.json file configuration.
Fixes:
- 
- 
-

### CODE REVIEW

**Comment by proable** [comment]
may i know where do i place this config to run?

**Comment by helloitu** [comment]
is a new file , when merged the request , uuse git pull , and then ./full-installer.sh

**Comment by proable** [comment]
i don't understand. can you tell me step by step how should it be done?

**Comment by helloitu** [comment]
the installer is not accepted , so isn't working for now . Just wait

**Comment by helloitu** [comment]
but if you want , https://github.com/helloitu/PokemonGo-Bot , clone this repo and execute ./full-installer.sh

**Comment by dezayner** [comment]
config.json.bak is not included in this merge, why ?

**Comment by proable** [comment]
is it possible for you to make a video tutorial?

**Comment by helloitu** [comment]
@dezayner  sorry , my bad.

**Comment by helloitu** [comment]
i learning about git.

File: full-installer.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+cd ..
+cd configs/
+cp config.json.bak  config.json
+echo "configuring ......"
+echo "----------------------"
+echo "Enter with you ptc login: "
+read ptcLog
+echo "-----------------------"
+echo "Enter with your ptc password: "
+read ptcPass
```
Comment (eriweb) [inline full-installer.sh]: consider adding the option `-s` to the `read` command, that will prevent it from displaying it while typing

File: full-installer.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+echo "Desenvolvido por Guilherme:) www.facebook.com/guilhermexyz"
+echo "----------------------------------------------------------"
+sudo su
```
Comment (eriweb) [inline full-installer.sh]: Bad practice to invoke a su shell, sudo the necessary commands instead

File: full-installer.sh (hunk 1 above)
Comment (helloitu) [inline full-installer.sh]: thx

**Pull request closed without merging**